src/main/java/com/karankumar/bookproject/ui/goal/DeleteGoalEvent.java

```diff
@@ -0,0 +1,11 @@
+package com.karankumar.bookproject.ui.goal;
+
+import com.karankumar.bookproject.backend.entity.ReadingGoal;
+public class DeleteGoalEvent {
+    // TODO: implement deleting reading goal
+    public static class DeleteEvent extends ComponentGoalEvent.GoalFormEvent {
```

Comment:
Could you please remove the inner class and make the current outer class extend `ComponentGoalEvent.GoalFormEvent`. Note that a constructor will be needed

Reply:
I've deleted this